### PR TITLE
tests(devtools): sync webtests

### PR DESCRIPTION
**Pull request by connorjclark**
ref https://chromium-review.googlesource.com/c/devtools/devtools-frontend/+/2902746

### CODE REVIEW

File: .github/workflows/devtools.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
           ${{ env.BLINK_TOOLS_PATH }}
           ${{ github.workspace }}/lighthouse/.tmp/chromium-web-tests/content-shells
         # The number is how many times this hash key was manually updated to break the cache.
-        key: ${{ runner.os }}-1-${{ hashFiles('third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/*.js', 'lighthouse/.github/workflows/devtools.yml', 'lighthouse/lighthouse-core/test/chromium-web-tests/*', 'lighthouse/clients/devtools-entry.js', 'lighthouse/clients/devtools-report-assets.js', 'lighthouse/build/build-bundle.js', 'lighthouse/build/build-dt-report-resources.js') }}
+        key: ${{ runner.os }}-2-${{ hashFiles('third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/*.js', 'lighthouse/.github/workflows/devtools.yml', 'lighthouse/lighthouse-core/test/chromium-web-tests/*', 'lighthouse/clients/devtools-entry.js', 'lighthouse/clients/devtools-report-assets.js', 'lighthouse/build/build-bundle.js', 'lighthouse/build/build-dt-report-resources.js') }}
```
Comment (connorjclark) [inline .github/workflows/devtools.yml:36]: i love that we have this but why isn't the `hashFiles` cachebusting when we tweak the files....

Comment (connorjclark) [reply .github/workflows/devtools.yml:36]: maybe paths are wrong? https://github.com/actions/runner/issues/556

Comment (connorjclark) [reply .github/workflows/devtools.yml:36]: wow https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#hashfiles

and it only accepts 1 arg....

**Review by paulirish — APPROVED** [review]

**Comment by mirul5764** [comment]
uwfcsdhe

**Comment by mirul5764** [comment]
https://chromium-review.googlesource.com/c/devtools/devtools-frontend/+/2902746

**Pull request merged**